src/azure-cli/azure/cli/command_modules/apim/_help.py

```diff
@@ -612,3 +627,30 @@
         az apim api schema wait --created --api-id MyApi --name MyApim --schema-id schemaId -g MyResourceGroup
     crafted: true
 """
+
+helps['apim deletedservice show'] = """
+type: command
+short-summary: Get soft-deleted Api Management service instances available for undelete by name.
+examples:
+  - name: Get a soft-deleted services with its name.
+    text: |
+        az apim deletedservice show --service-name MyApim --location westus
+"""
+
+helps['apim deletedservice list'] = """
+type: command
+short-summary: Lists all soft-deleted Api Management services instances available for undelete for the given subscription.
```

Comment:
Please use the first-person voice in the help message
```suggestion
short-summary: List all soft-deleted Api Management services instances available for undelete for the given subscription.
```